core/kazoo_config/src/kapps_config.erl

```diff
@@ -31,6 +31,8 @@
 -export([get_non_empty/2, get_non_empty/3, get_non_empty/4]).
 -export([get_ne_binary/2, get_ne_binary/3, get_ne_binary/4]).
 
+-export([get_inherited/4, get_inherited/5]).
```

Comment:
@mark2600 i don't think this should go here. i think it will fit better in `kz_account` since we have the `fetch` method there and you are looking for values in the account document not in configuration